src/routes/auth.route.ts

```diff
@@ -1,21 +1,71 @@
 import express, { Request, Response } from "express";
+import superagent from "superagent";
 
 import { register } from "../services/auth.service";
 
 const authRouter = express.Router();
 
-// authRouter.route("/twitter").get(
-// post request to Twitter API
-// receive redirectUri in response
-// redirect your user to received redirectUri res.redirect()
-// );
+const redirectUri = `http://localhost:${process.env.PORT}/api/auth/twitter-callback`;
+const twitterClientId = process.env.TWITTER_CLIENT_ID;
+const twitterClientSecret = process.env.TWITTER_CLIENT_SECRET;
+const state = {};
+const codeChallenge = "sdSD939fFFsfdfdskojilmsionjfje";
```

Comment:
```suggestion
const codeChallenge = "sdSD939fFFsfdfdskojilmsionjfje";
```
Why was it written strait in code. May be move it to process.env

Reply:
is done